pages/common/routers/success.js

```diff
@@ -1,8 +1,24 @@
+const { merge, get } = require('lodash');
 const { Router } = require('express');
+const successContent = require('../content/success-messages');
 
-module.exports = ({ model = 'model' } = {}) => {
+module.exports = ({
+  model = 'model',
+  licence = 'pel',
+  type = 'amendment'
+} = {}) => {
   const app = Router();
 
+  app.use((req, res, next) => {
+    if (req.model && req.model.status) {
+      type = req.model.status === 'active' ? 'amendment' : 'application';
+    }
+    const success = get(successContent, `${licence}.${type}.resubmitted`);
```

Comment:
Hard-coded `resubmitted` strikes me as odd.

Reply:
yeah - it could be passed in, but it is resubmitted 100% of the time

Reply:
the logic for displaying task success messages lives elsewhere

Reply:
also resubmitted === submitted